config/defaults.ini.php

```diff
@@ -0,0 +1,457 @@
+<?php return <<<INI
+
+; This file declares the default values for all the config values of Friendica.
+; Please don't edit this file as its content may change in the upcoming versions.
+
+[jabber]
```

Comment:
Could you change the order of the sections in the config file? E.g. the database settings are more important then the jabber debug settings.

Reply:
Yeah, I took the order in the former `doc/htconfig.md` file but it can be changed.